README.md

```diff
@@ -145,6 +145,18 @@ Curated language templates:
 
 Read more on [community templates here](guide/TEMPLATE.md).
 
+**Templates store**
+
+The template store has templates from openfaas and openfaas-incubator organizations with openfaas templates being the official ones.
+
+In order to see what templates are available in the store type `faas-cli template store list`.
+
+> Note: You can set your own custom store location with `--url` flag or set `OPENFAAS_TEMPLATE_STORE_URL` environmental variable
+
+To pull templates from the store just write the name of the template you want, you can pull couple of templates at once `faas-cli template store pull go golang-http python3-debian ...`
```

Comment:
I didn't mention multiple-pulls in the same command in the original issue. I think we could probably reduce complexity by removing this behavior.

The main use-case is for templates to be pulled from the default location or a special signal template repo to be pulled in for a project.

Reply:
Ok yeah I will remove the feature

Reply:
Feature removed ✔️